tools/unicode_ranges.py

```diff
@@ -229,7 +231,7 @@ def ranges(i):
     :return: List of ranges.
     """
 
-    for a, b in itertools.groupby(enumerate(i), lambda (x, y): y - x):
+    for _, b in itertools.groupby(enumerate(i), lambda (x, y): y - x):
```

Comment:
It's correct, a throwaway variable name to indicate that part of the result is being ignored.